### PR TITLE
Use sanity data for company

### PR DESCRIPTION
**Pull request by mirisuzanne**
Testing out live sanity data in a branch…

- [x] for company/site data
- [x] for fully-dynamic pages (prison arts)
- [x] for mixed pages (about & contact)
- [x] for shows
- [x] for ticket sales with stripe

Feel free to test the checkout form, and cancel out once you reach the Stripe payment system.

### CODE REVIEW

File: content/_data/articles.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  `).then(data => {
+    data.map(page => {
+      page.body = page.body ? toMarkdown(page.body) : '';
+      page.hero = heroAlt(page.hero);
+      return page;
+    });
+
+    return _.groupBy(data, item => item.category);
+  });
+};
```
Comment (jgerigmeyer) [inline content/_data/articles.js]: This should work fine, but usually I would expect to see either `async/await` _or_ `.then()` syntax -- in this case it's redundant. Anything after `await` is essentially running within a `.then(...)` context. For example, I might expect e.g.:

```suggestion
module.exports = async function() {
  const data = await client.fetch(groq`
    *[_type == "article"
      && !(_id in path("drafts.**"))
    ]{
      title,
      subtitle,
      summary,
      body,
      "slug": slug.current,
      "tags": tags[]->title,
      "date": _createdAt,
      "category": category->title,
      ${hero},
    }
  `);

  const pages = data.map(page => {
    page.body = page.body ? toMarkdown(page.body) : '';
    page.hero = heroAlt(page.hero);
    return page;
  });

  return _.groupBy(pages, page => page.category);
};
```

File: content/_data/articles.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "date": _createdAt,
+      "category": category->title,
+      ${hero},
+    }
+  `).then(data => {
+    data.map(page => {
+      page.body = page.body ? toMarkdown(page.body) : '';
+      page.hero = heroAlt(page.hero);
+      return page;
+    });
```
Comment (jgerigmeyer) [inline content/_data/articles.js]: `Array.map()` doesn't alter the original array, and the returned value isn't being used -- is this block doing anything currently?

File: functions/success.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+const apiKeys =
+  environment !== "production"
+    ? environmentKeys.other
+    : environmentKeys.production;
+
+const stripe = require("stripe")(apiKeys.STRIPE_KEY);
+const client = require('../utils/sanityClient');
+
+exports.handler = async function (event, context) {
```
Comment (jgerigmeyer) [inline functions/success.js:22]: When is this `success` function called? I don't see a clear reference somewhere.

**Review by jgerigmeyer — COMMENTED** [review]
@mirisuzanne I don't know anything about Stripe, or Groq, or Sanity, so... not sure how helpful I can be? 😉

I don't see any major issues in the JS, other than the one usage of `.map()` and some redundant `async/await` vs Promise/`.then()` code. 👍

File: content/_data/articles.js (hunk 1 above)
Comment (jgerigmeyer) [reply content/_data/articles.js]: Same goes for most other uses of `await` throughout this PR.

Comment (mirisuzanne) [reply content/_data/articles.js]: Thank you! I still haven't totally got my mind around async/await - this is real helpful.

File: functions/success.js (hunk 3 above)
Comment (mirisuzanne) [reply functions/success.js:22]: There's a webhook in the Stripe setup that triggers the success function when a sale is complete. (there's also a CLI tool for triggering & listening to webhook events in development, to test that this works)

**Pull request merged**